frontend/src/shared/ui/file-upload/file-uploader.module.css

```
@@ -2,31 +2,28 @@
-  border-radius: var(--radius-lg);
+  border-radius: 8px;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
   padding: 40px;
-  background: rgba(255, 255, 255, 0.5);
-  backdrop-filter: blur(8px);
-  -webkit-backdrop-filter: blur(8px);
+  background: #ffffff;
+  cursor: pointer;
```

Stylelint (17.7.0)

[error] Duplicate property "cursor" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)